smart-contracts/contracts/utils/tests/SelfDestructible.sol

```diff
@@ -0,0 +1,5 @@
+contract SelfDestructible {
```

Comment:
?

Reply:
That's used in the tests to bypass the `receive` function check that 
rejects value sent directly - per the PoC from the audit.